### ISSUE TITLE
Activation function in neural network

### ISSUE THREAD

**Issue by xujiaze13** [chris-nemeth/sgmcmc-review-paper#2]
https://github.com/chris-nemeth/sgmcmc-review-paper/blob/dbdb0ecfc13dce0a2e8160168b821ca1cc5aa27d/bayesian_neural_network/bnn_model.R#L4

Why the activation function in the hidden layer is `softmax`?
Should it be the `sigmoid` function?
